packages/block-library/src/archives/index.php

```diff
@@ -60,7 +60,7 @@ function render_block_core_archives( $attributes ) {
 
 		$label = esc_html( $label );
 
-		$block_content = '<label class="screen-reader-text" for="' . $dropdown_id . '">' . $title . '</label>
```

Comment:
Do we know why this has been added in the first place? 

Also, if we decide to remove it, we should remove it from the editor as well. Also, this is the only place where this class is used for Core blocks which means we need to remove the styles defined in `common.scss` for it as well.

Reply:
> Do we know why this has been added in the first place?

When looking at the PR in which this block has been introduced (#7949) I can't find a context about why this class was added in the first place.
I assume it's because they want to hide it (?) but still making it available in the screen reader tree.

Pinging @talldan @mcsf for more details.

Reply:
The original PR matched the widget I believe

Reply:
> Also, if we decide to remove it, we should remove it from the editor as well.

It isn't what happens when I remove the class in `/block-library/src/archives/index.php`? It is removed in the editor and in the frontend.

Reply:
> The original PR matched the widget I believe

So it may be to avoid duplicate with the widget title 🤔

Reply:
> Also, this is the only place where this class is used for Core blocks which means we need to remove the styles defined in `common.scss` for it as well.

It is also used for the Core Search block https://github.com/WordPress/gutenberg/blob/8be7a8d0ee338c9d4668dab08733266a325f713f/packages/block-library/src/search/index.php#L50

Reply:
> The original PR matched the widget I believe

It's been a long time, but I assume it was following prior art. I don't know how relevant that label still is.

Reply:
@joedolson , do you think there's an accessibility context for this `screen-reader-text` class? Thanks :)

Reply:
I also don't recall. It was one of the first PRs I contributed, but I mostly took over from a contributor that wasn't able to finish it.